.tekton/rhtap-cli-pull-request.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
       event == "pull_request"
       && target_branch == "main"
       && files.all.exists(x, !x.matches('^(?:docs|.github|.vscode|.ci)/|\\.md$|^(?:.*/)?(?:\\.gitignore|OWNERS|PROJECT|LICENSE.txt|.golangci.yaml|.goreleaser.yaml|.snyk|.dockerignore|yamllint.yaml)$'))
-  creationTimestamp: null
+  creationTimestamp:
```

Comment:
This look strange to me. I don't think an empty value is valid YAML.

Reply:
I tried few online yaml validators and according to those this is valid.
I also checked few repos managed by Konflux and they merged the same change. For example: 
https://github.com/konflux-ci/integration-service/blob/main/.tekton/integration-service-push.yaml#L10
https://github.com/konflux-ci/release-service/blob/main/.tekton/release-service-push.yaml#L10

Reply:
I'll approve the PR then.